### DISCUSSION TITLE
Spinning up a full node on testnet, genesis unable to be parsed

### DISCUSSION THREAD

**Discussion by jalw0tpyrc** [MystenLabs/sui#11004] [Q&A]
Hi, 
Trying to spin up a full node on the testnet and I'm getting 
`fullnode_1  | Error: Unable to parse Genesis from genesis.blob`

I'm downloading the Genesis from https://github.com/MystenLabs/sui-genesis/blob/main/testnet/genesis.blob 

Any help would be greatly appreciated thanks.

**Comment by after-ephemera** [comment]
Try downloading the blobs found [in the docs](https://docs.sui.io/build/fullnode#setting-up-a-full-node-from-source) and see if that works better for you.

**Reply by jalw0tpyrc** [reply]
Hey, 
The docs shows same URL ... No success. Anything else?

**Comment by Godeanstreet** [comment]
0x0908be3fc48fcba7340ad640c3f78e35264ee463c8891d16b5e7f2522634a7a3
